src/sections/Community/Members-grid/membersGrid.style.js

```diff
@@ -159,11 +158,20 @@ export const MembersGridWrapper = styled.div`
   .members-profile-cards {
     z-index: -1 !important;
   }
+  #Badge_img {
```

Comment:
We don't use ids (#). Please follow suit with the rest of the pre-established styling convention.

Reply:
@PRANJALRANA11 did you incorporate this feedback?

Reply:
I will commit it by today